en/user_guide/node.rst

```diff
@@ -0,0 +1,88 @@
+Node
+====
+
+Languages
+---------
+
+A node has several languages defined during the configuration of the Back Office.
+The node language being edited can be changed by clicking on the language tab. If this node has no data for this language, it will be created.
+
+.. image:: ../../images/languages.png
+
+Workflow
+--------
+
+Nodes have a status and can have a validation workflow, which is the series of steps to achieve for publication of a node before it can be displayed on Front Office.
+Status of a node is linked to language of this one.
+Node have an initial status for this creation and to be able to display on Front Office the node has to have a final status.
+
+.. image:: ../../images/workflow.png
+
+To know more about status see also `workflow`_.
+
+Versionning
+-----------
+
+A node have several versions.
+The node version is linked to language of this one.
+To create a new version of a node click on the "new version" button.
+
+.. image:: ../../images/new_version.png
+
+The latest version of the node is duplicated and the node is set to the initial status.
+
+Then find all the versions of a node in the drop-down list.
+
+.. image:: ../../images/version_list.png
+
+The latest version of the node in the final status will be the one displayed in Front Office.
+
+Blocks
+------
+
+Open Orchestra allows to add blocks in nodes.
+The visible list of blocks during the edition of a node is defined during the site configuration.
+
+See also `node configuration`_ and `site configuration`_.
+
+There are two types of blocks :
+
+* Transverse block.
+* Not transverse block.
```

Comment:
I would rather explain in one sentence that a node can use transverse and not transverse blocks. That the transverse ones are not editable in other nodes, etc.